src/app/navbar/navbar.component.css

```diff
@@ -97,6 +97,9 @@
   #navcontainer, #side-menu {
     margin-top: 8px;
   }
+  .top-nav {
+    width: 101.2%;
```

Comment:
I don't think this will help. the outer divs which is leaving whitespace is in resultscomponent and appcomponent, try looking why they are not taking full length?
Ideally a width of 100% should work, why is 100% not a good solution?